test-forms/airbnb_signup.html

```diff
@@ -1,424 +1,282 @@
+                    won’t be shared with other people who use Airbnb.</div>
+            </div>
+        </fieldset>
     </div>
     <div class="_vp4g0q">
-        <div>
-            <div class="_1jj61m9"><label class="_1yw7hpv" for="email-signup-email">
-                <div class="_144jzckf">
-                    <div class="_11hx67x">Email</div>
-                </div>
-                <div dir="ltr">
-                    <div class="_fywymp7"><input aria-describedby="email-signup-email-InputField-help" autocomplete="off" class="_yrzsibu"
-                                                 data-manual-scoring="emailAddress" data-testid="email-signup-email"
-                                                 id="email-signup-email" inputmode="email"
-                                                 name="user[email]" placeholder="Email"
-                                                 >
-                    </div>
-                </div>
-            </label></div>
-            <div class="_klarpw">
-                <div class="_pxnjg" data-testid="email-signup-email-InputField-help"
-                     id="email-signup-email-InputField-help">We'll email you trip confirmations and receipts.
+        <fieldset class="c18cwni1 atm_3f_glywfm atm_gi_idpfg4 atm_l8_idpfg4 dir dir-ltr">
+            <legend data-base-field-group-label="true"
+                class="l101oo45 atm_7l_jt7fhx atm_bx_48h72j atm_c8_2x1prs atm_g3_1jbyh58 atm_fr_11a07z3 atm_cs_10d11i2 atm_lj_idpfg4 atm_li_idpfg4 atm_gq_exct8b dir dir-ltr">
+                <h2 tabindex="-1"
+                    class="hpipapi atm_7l_1kw7nm4 atm_c8_1x4eueo atm_cs_1kw7nm4 atm_g3_1kw7nm4 atm_gi_idpfg4 atm_l8_idpfg4 atm_kd_idpfg4_pfnrn2 dir dir-ltr"
+                    elementtiming="LCP-target">Contact info</h2>
+            </legend>
+            <div>
+                <div data-base-input-root="true" class="_12huvry"><label class="_1yw7hpv" for="email-signup-email">
+                        <div
+                            class="l1bm6uz3 atm_mk_stnw88 atm_tk_f6fqlb atm_fq_1fwxnve atm_n3_1fwxnve atm_gi_idpfg4 atm_l8_idpfg4 atm_mj_glywfm atm_bx_48h72j atm_c8_2x1prs atm_g3_1jbyh58 atm_fr_11a07z3 atm_7l_1he744i atm_uc_1pr5qnm atm_tw_xchc94 l6fh4mc atm_vy_1y5hjwv atm_tr_1y95dkk atm_cs_6adqpa dir dir-ltr">
+                            <div
+                                class="l12j3uvm atm_j3_1osqo2v atm_ks_15vqwwr atm_sq_1l2sidv atm_vv_1q9ccgz dir dir-ltr">
+                                Email</div>
+                        </div>
+                        <div class="_3qiqzub"><input inputmode="email" aria-disabled="false" name="user[email]"
+                                data-testid="email-signup-email" class="_1p9gixpg" id="email-signup-email"
+                                autocomplete="email" type="email" aria-describedby="email-signup-email-InputField-help"
+                                placeholder="Email" value="dbajpeyi112@dmail.com"
```

Comment:
It's best to remove your personal email.
```suggestion
                                placeholder="Email" value=""
```